rules.d/60-persistent-storage.rules

```diff
@@ -12,8 +12,11 @@ KERNEL!="loop*|mmcblk*[0-9]|msblk*[0-9]|mspblk*[0-9]|nvme*|sd*|sr*|vd*|xvd*|bcac
 # ignore partitions that span the entire disk
 TEST=="whole_disk", GOTO="persistent_storage_end"
 
-# for partitions import parent information
-ENV{DEVTYPE}=="partition", IMPORT{parent}="ID_*"
+# for partitions import parent ID_* information, except ID_FS_*
```

Comment:
can you add a slightly longer comment here. It' not obvious why to exclude ID_FS_*, and we should say so here